src/command_modules/azure-cli-component/azure/cli/command_modules/component/commands.py

```diff
@@ -3,11 +3,11 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-#pylint: disable=line-too-long
```

Comment:
I recommend leaving this in place. Most command modules disable "line too long" on `commands.py` and `params.py`. Instead I would recommend replacing as much of the common string gibberish (azure.cli.command_modules.component.custom#) with a format string:
`custom_path='azure.cli.command_modules.component.custom#{}'` 
This can then be used in place of the long ugly string to make the lines both shorter (though still likely "too long" by Pylint standards) and easier to read.